docs/admin/accessing-the-api.md

```diff
@@ -1,72 +1,151 @@
 ---
 ---
 
-This document describes what ports the Kubernetes apiserver
-may serve on and how to reach them.  The audience is
-cluster administrators who want to customize their cluster
-or understand the details.
+This document describes how access to the Kubernetes API is controlled.
 
-Most questions about accessing the cluster are covered
-in [Accessing the cluster](/docs/user-guide/accessing-the-cluster).
+# Overview
 
+Users [access the API](/docs/user-guide/accessing-the-cluster) using `kubectl`,
+client libraries, or by making REST requests.  Users include human users and
+[Kubernetes Service Accounts](http://kubernetes.io/docs/user-guide/service-accounts/).
+When a request reaches the API, it goes through several stages, illustrated in the
+following diagram:
 
-## Ports and IPs Served On
+<img src="/images/docs/admin/accessing-the-api.svg">
```

Comment:
not sure how image references are supposed to be specified... absolute or relative?